packages/node/src/builders/execute/execute.impl.ts

```diff
@@ -54,16 +55,27 @@ export function nodeExecuteBuilderHandler(
       return startBuild(options, context).pipe(
```

Comment:
I actually think an easier way to handle this feature is to pass through `watch` as an override to build. That will complete the `build` observable thus not restarting the process. Can you please try that solution?

Reply:
I think that passing the `config.watch` to that `watch`overrides the webpack configuration. Obviously when the `config.watch` is false, does not watch for changes and does not restart which is a problem.

How I checked:
```
scheduleTargetAndForget(context, target, {
          watch: options.watch,
        }) as Observable<NodeBuildEvent>
```